test.py

```diff
@@ -51,13 +51,15 @@ def on_receive_message(topic, message):
 tls_options = None
 if args.cert or args.key or args.root_ca:
     if args.cert:
-        assert(args.key)
-        tls_options = io.TlsContextOptions.create_client_with_mtls(args.cert, args.key)
+        assert (args.key)
+        tls_options = io.TlsContextOptions.create_client_with_mtls_from_path(args.cert, args.key)
     else:
         tls_options = io.TlsContextOptions()
 
     if args.root_ca:
-        tls_options.override_default_trust_store(ca_path=None, ca_file=args.root_ca)
+        with open(args.root_ca, mode='rb') as ca:
+            rootca = ca.read()
+        tls_options.override_default_trust_store(rootca)
```

Comment:
why not use `override_default_trust_store_from_path`?

Reply:
To exercise both paths and the null termination, and act as an example